### DISCUSSION TITLE
ignoreDistance

### DISCUSSION THREAD

**Discussion by CowboyPilotUSA** [antihax/RadioNotifications#1] [General]
I'm a bit uncertain about how exactly this works? Can you provide details about this setting?

**Comment by antihax** [comment]
Notifications beyond `(maxDistance * baseRadioMultiplier) + ignoreDistance` are not queued onto a radio to be played. It is to prevent overfilling a queue with messages. 

You would want this set to a reasonable distance a player would be unable to travel in about 30 seconds. With helicopters and cars, they can move a bit of distance.
